From 500deb2a992de1236f5b91dc77e0d415d05345c3 Mon Sep 17 00:00:00 2001
From: Robin Gong <b38343@freescale.com>
Date: Wed, 11 Mar 2015 15:15:13 +0800
Subject: [PATCH] MLK-10404-2 ARM: dts: imx6dl: support ecspi DMA on i.mx6dl

Enable ecspi DMA support on i.mx6dl since there is official workaround for
TKT238285.

Signed-off-by: Robin Gong <b38343@freescale.com>
(cherry picked from commit 99f3b9a581af1b9e5a3fa05ae1ad68e165ce8b2c)
---
 arch/arm/boot/dts/imx6dl.dtsi | 20 ++++++++++++++++++++
 1 file changed, 20 insertions(+)

diff --git a/arch/arm/boot/dts/imx6dl.dtsi b/arch/arm/boot/dts/imx6dl.dtsi
index 66bf5b0..84b9a16 100644
--- a/arch/arm/boot/dts/imx6dl.dtsi
+++ b/arch/arm/boot/dts/imx6dl.dtsi
@@ -175,6 +175,26 @@
 	};
 };
 
+&ecspi1 {
+	dmas = <&sdma 3 7 1>, <&sdma 4 7 2>;
+	dma-names = "rx", "tx";
+};
+
+&ecspi2 {
+	dmas = <&sdma 5 7 1>, <&sdma 6 7 2>;
+	dma-names = "rx", "tx";
+};
+
+&ecspi3 {
+	dmas = <&sdma 7 7 1>, <&sdma 8 7 2>;
+	dma-names = "rx", "tx";
+};
+
+&ecspi4 {
+	dmas = <&sdma 9 7 1>, <&sdma 10 7 2>;
+	dma-names = "rx", "tx";
+};
+
 &ldb {
 	compatible = "fsl,imx6dl-ldb", "fsl,imx53-ldb";
 
-- 
1.8.0

